qutebrowser/config/configdata.py

```diff
@@ -360,6 +360,10 @@
          SettingValue(typ.Bool(), 'false'),
          "Hide the tabbar if only one tab is open."),
 
+        ('perm-hide',
```

Comment:
I think `perm-` is a bit confusing. I'd prefer `always-hide`. Alternatively, renaming `auto-hide` to `hide-auto` and `perm-hide` to `hide-always` would sort them correctly.

What's your opinion on this?